pkg/scheduler/plugins/nodenumaresource/scoring.go

```diff
@@ -89,6 +92,37 @@ func (p *Plugin) Score(ctx context.Context, cycleState *framework.CycleState, po
 	return p.scorer.score(requested, allocatable, framework.NewResource(resourceOptions.requests))
 }
 
+func (p *Plugin) scoreWithAmplifiedCPUs(cycleState *framework.CycleState, state *preFilterState, pod *corev1.Pod, nodeInfo *framework.NodeInfo, topologyOptions TopologyOptions) (int64, *framework.Status) {
+	quantity := state.requests[corev1.ResourceCPU]
+	if quantity.IsZero() {
+		return 0, nil
```

Comment:
I think we should call the default scorer here rather than return 0 to keep the same behavior as the `NodeResourceFit` plugin.